imagepuller/internal/service/stubs_test.go

```diff
@@ -19,14 +21,36 @@ type stubStore struct {
 	putLayerDigest digest.Digest
 	putLayerErr    error
 
+	layers map[string]struct{}
+
```

Comment:
nit: this is fixing stub and fake implementations, but I think we can live with it.